<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>1196</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="play1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>530</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Video 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="video1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="convertedVideo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>600</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playc">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>870</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play the converted video</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fade">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fade</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="cut">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wipe">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>680</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wipe</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="scale">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>720</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="picInPic">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>760</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture in Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>820</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pausec">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>920</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause the converted video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Video 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="video2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pause2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>530</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Video 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Video 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAssignment_1_SOEN_6761">
    <property name="title">
     <string>Assignment 1 SOEN 6761</string>
    </property>
   </widget>
   <addaction name="menuAssignment_1_SOEN_6761"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
